### ISSUE TITLE
Multi-pages form submission: can't go to next page because of errors on next page (which hasn't been filled yet)

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#1373]
### Describe the bug

Here's the reduced test case:
```
{% set form = craft.formie.forms.handle(craft.app.request.param('handle')).one() %}
{% set submission = craft.formie.submissions.id(craft.app.request.getParam('id')).isIncomplete('*').one() %}
{% do form.setSubmission(submission) %}
{{ craft.formie.renderForm(form) }}
```
So the test URL looks like: `/test?handle=myForm&id=123456`

### Steps to reproduce

1. Login as admin;
2. Use a multi-pages form with required fields on each page with setup of `Submission Method` to `Page Reload`;
3. Open an existing incomplete multi-pages form submission that have just its first page filled in (using `/test?handle=myForm&id=123456`);
4. Submit the first page, and get required fields errors of second page in `formie.log`;
5. Get stuck on the first page instead of being redirected to the second page;

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.4.5

### Plugin version

dev-craft-4 as 2.0.25

### Multi-site?

Yes

**Comment by engram-design** [comment]
Looks to be due to [my change](https://github.com/verbb/formie/issues/758#issuecomment-1487602666) which I'm going to revert. I think it's better to control the current submissions state with `setSubmission()` and `resetCurrentSubmission()` to clear the session, rather than messing around with `sessionKey`.

To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.25.1"`

**Comment by romainpoirier** [comment]
Thank you, now it works: I don't get the next page's required fields errors.

However, I'm still stuck on the first page on submit when I fill a new submission without yet knowing the `id` to use in parameter. I'm going to add additional remarks on the initial #758 issue.

**Comment by engram-design** [comment]
Fixed in [2.0.27](https://github.com/verbb/formie/releases/tag/2.0.27)
